server/src/test/java/org/cloudfoundry/identity/uaa/zone/ZoneAwareClientSecretPolicyValidatorTests.java

```diff
@@ -39,6 +39,8 @@ void setUp() {
     @Test
     void testEmptyClientSecret() {
         zone.getConfig().setClientSecretPolicy(defaultPolicy);
+        validator.validate(TEST_SECRET_1);
```

Comment:
yes it is empty and the test was wrong before... and this is independent from all other changes related to empty secret.

Here we simply have the issue, that via REST the client creation with empty secret was never possible.... or lets says... since this valdation here... because in YAML you have client.secret.policy ...minLenght: 0, but in code it then wants 1 as mimimum.

Reply:
I understand that. What I'm saying is that I'm looking at this test called `testEmptyClientSecret` that you are modifying, and I cannot figure out where the empty client secret actually is.

Could you either rename the constant to be called `EMPTY_SECRET` or something so that we understand what the test is doing?

Reply:
rename TEST_EMPTY_SECRET